Include/internal/pycore_bitutils.h

```diff
@@ -82,6 +84,31 @@ _Py_bswap64(uint64_t word)
 }
 
 
+// Population count: count the number of 1's in 'u'
+// (number of bits set to 1)
+static inline int
+_Py_popcount32(uint32_t u)
+{
+#if defined(__clang__) || defined(__GNUC__)
+
+#if SIZEOF_INT == 4
+    return __builtin_popcount(u);
+#elif SIZEOF_LONG == 4
+    return __builtin_popcountl(u);
+#else
+#  error "unsupported configuration"
```

Comment:
Why not just use the fallback implementation in this case?

Reply:
I would prefer to get a build error when we fail to get a builtin function whereas we could use one. In practice, nobody should see this error :-)

Reply:
Well, I rewrote the code to avoid this `#error`.